### ISSUE TITLE
Can't draw a polygon with antialiasing although interpolate=True

### ISSUE THREAD

**Issue by mmagnuski** [psychopy/psychopy#4293]
Hi,
I've had this issue on various computers so far and would like to cooperate with you (psychopy devs) to find the root of the problem. :)
I'm trying to draw an antialiased circle, but I keep getting pixelated edges although I set `interpolate=True` (which is the default anyway):

```python
from psychopy import visual, event, core

window = visual.Window()
circle = visual.Polygon(window, edges=26, radius=100, interpolate=True,
                        units='pix', fillColor=[1, 1, 1])

circle.draw()
window.flip()
event.waitKeys(keyList=['space'])
core.quit()
```
I see this circle displayed:
![obraz](https://user-images.githubusercontent.com/8452354/135438893-589b76b1-c4c5-48a8-bc99-c0dcdc7dc9dc.png)
you can see it is not antialiased. The same happens when creating the circle in Builder.

I'm on Windows 10, Psychopy 2021.2.2.

**Comment by mmagnuski** [comment]
my graphics card is `NVIDIA GeForce GTX 1660`. Setting `useFBO=False` when creating the window does not help.

related issues:
#753
#594

https://discourse.psychopy.org/t/drawing-multiple-anti-aliased-lines/2593/10
https://discourse.psychopy.org/t/visual-stimulus-edges-appear-blurred/19535

**Comment by mmagnuski** [comment]
Ok, creating the window in this way:
```python
window = visual.Window(useFBO=False, multiSample=True, numSamples=16)
```
does make antialiasing work.

**Comment by mdcutone** [comment]
Right, we don't yet support multisample FBOs (unless your are using VR) so you need to use the default back buffer with multisampling enabled. Setting the shape's outline color might make it work since it will draw an outline, lines are antialiased by the driver so multisampling may not be needed.
